structured/structured.proto

```diff
@@ -69,6 +69,14 @@ message IndexDescriptor {
   repeated uint32 column_ids = 5 [(gogoproto.customname) = "ColumnIDs"];
 }
 
+// PrivilegeDescriptor represents the sets of privileges on a descriptor.
+message PrivilegeDescriptor {
```

Comment:
we already have a bunch of privileges in parser. additionally, this matches the naming (see IndexDescriptor and ColumnDescriptor)